_data/patterns/race.yml

```diff
@@ -0,0 +1,88 @@
+legend: |
+  What is your race and/or ethnicity?
+hint: |
+  Select all that apply
```

Comment:
I think a period is missing after "Select all that apply" - is that intentional?

Reply:
I left it off because the other example on our site that uses "Select all that apply" (See: the [preferred language pattern](https://designsystem.digital.gov/patterns/select-a-language/language-preferences/)) does not include a period. Let me know if you'd like me to add the period in.